composer.json

```diff
@@ -19,14 +19,16 @@
         "composer/installers": "^2.1",
         "cweagans/composer-patches": "^1.7",
         "drupal/admin_toolbar": "^3.3",
+        "drupal/config_split": "^2.0@RC",
         "drupal/core-composer-scaffold": "^10.0",
         "drupal/core-recommended": "^10.0",
         "drupal/monolog": "^3.0@beta",
         "drupal/simplei": "^2.0@beta",
         "drush/drush": "^11.4",
         "vlucas/phpdotenv": "^5.4",
         "webflo/drupal-finder": "^1.2",
-        "wunderio/drupal-ping": "^2.4"
+        "wunderio/drupal-ping": "^2.4",
+        "wunderio/updates_log": "^2"
```

Comment:
> I noticed there's this profile web/profiles/custom/basic/basic.info.yml Is it supposed to be used during fresh Drupal projects? Should we enable the updates_log there? It had one bug that it could not find CKEditor atm.

Very good question, @hkirsman! I think we could even remove this custom`basic` install profile since we don't use install profiles in our projects. And the [Drupal Recipes Initiative](https://www.drupal.org/about/core/strategic-initiatives-distributions-and-recipes) sums up the reasons nicely:

> Historically this functionality has been served by install profiles and distributions, but both of those iterations of the problem have some flaws: 
> 
> They are difficult to keep updated
> They cannot be added after starting your project
> They can't be removed easily
> They can't be mixed and matched with other sets of functionality.

Reply:
We're using [recipes](https://www.drupal.org/project/distributions_recipes) in our https://github.com/wunderio/next-drupal-starterkit template and it seems to be working well. Perhaps we should introduce it here as well?